x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/GetDataFrameAnalyticsStatsAction.java

```diff
@@ -173,6 +189,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             public XContentBuilder toUnwrappedXContent(XContentBuilder builder) throws IOException {
                 builder.field(DataFrameAnalyticsConfig.ID.getPreferredName(), id);
                 builder.field("state", state.toString());
+                if (analyticsProgressPercentage != null) {
+                    builder.field("analytics_progress_percentage", analyticsProgressPercentage);
```

Comment:
Any feedback about the name of this would be welcome here

Reply:
Since `RecoveryState` uses `percent`, it would be more consist to use `analytics_progress_percent` here instead of `analytics_progress_percentage`.